server/server.go

```diff
@@ -2727,7 +2728,9 @@ func (s *Server) acceptConnections(l net.Listener, acceptName string, createFunc
 		}
 		tmpDelay = ACCEPT_MIN_SLEEP
 		if !s.startGoRoutine(func() {
+			s.reloadMu.RLock()
```

Comment:
This for all connections?

Reply:
Yes, as we can have OCSP configured on any connection type. We don't know that the problem was unique to gateways.